data/json/materials.json

```diff
@@ -45,25 +68,28 @@
   },
   {
     "type": "material",
-    "ident": "acidchitin",
-    "name": "Biosilicified Chitin",
-    "density": 15,
-    "specific_heat_liquid": 4.186,
-    "specific_heat_solid": 2.108,
-    "latent_heat": 333,
-    "bash_resist": 3,
-    "cut_resist": 4,
-    "acid_resist": 13,
-    "fire_resist": 2,
-    "elec_resist": 2,
-    "chip_resist": 10,
-    "dmg_adj": [ "scratched", "cut", "cracked", "shattered" ],
-    "bash_dmg_verb": "cracked",
-    "cut_dmg_verb": "chipped",
+    "ident": "bean",
+    "name": "Bean",
```

Comment:
`Bean` should be earlier than `Biosilicified Chitin`.

Reply:
I ordered them by ID, not name (see the pull request solution section for more details)

Reply:
Well, in that case it's ok. Though I'd personally prefer sorting by their name.

Reply:
I agree, and I did that initially, but then I noticed that other files like armor.json also sort by item ID